src/electionguard_cli/e2e_steps/election_record_step.py

```diff
@@ -0,0 +1,52 @@
+from shutil import make_archive
+from os.path import splitext
+import tempfile
+import click
+from electionguard_cli.cli_models import BuildElectionResults, E2eSubmitResults
+from electionguard_cli.cli_models.e2e_decrypt_results import E2eDecryptResults
+from electionguard_cli.cli_models.e2e_inputs import E2eInputs
+from electionguard_cli.e2e_steps.e2e_step_base import E2eStepBase
+from electionguard.constants import get_constants
+
+from electionguard_tools.helpers.export import export_record
+
+
+class ElectionRecordStep(E2eStepBase):
+    """Responsible for publishing an election record after an election has completed."""
+
+    REMOVE_RAW_OUTPUT = True
+    REMOVE_ZIP_OUTPUT = True
+    COMPRESSION_FORMAT = "zip"
```

Comment:
If this doesn't have a `_` leading it can be changed. I think this may be able to be changed so this could just be a variable. 
`CompressionFormat`